# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/mp4/SampleIterator.h
# Commit: 93cf96e033d0
# Full Hash: 93cf96e033d0d7c13b66a279bb4cb1f7138dd473
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-05-31 21:43:54
# Description:
#   Bug 1833896 - Consistently use uint32_t to store timescale until it reaches the TimeUnit constructor, which uses int64_t. r=media-playback-reviewers,karlt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179363
# ==============================================================================

diff -r 5b7624f7163b -r 93cf96e033d0 dom/media/mp4/SampleIterator.h
--- a/dom/media/mp4/SampleIterator.h	Wed May 31 09:35:17 2023 +0000
+++ b/dom/media/mp4/SampleIterator.h	Wed May 31 09:36:03 2023 +0000
@@ -99,7 +99,7 @@
   };
 
   MP4SampleIndex(const mozilla::IndiceWrapper& aIndices, ByteStream* aSource,
-                 uint32_t aTrackId, bool aIsAudio, int32_t aTimeScale);
+                 uint32_t aTrackId, bool aIsAudio, uint32_t aTimeScale);
 
   void UpdateMoofIndex(const mozilla::MediaByteRangeSet& aByteRanges,
                        bool aCanEvict);